.github/CONTRIBUTING_GUIDELINES.md

```diff
@@ -31,28 +31,47 @@
 
 - Install `nodejs` and `npm` on your local machine. For [windows](https://www.geeksforgeeks.org/installation-of-node-js-on-windows/), [linux](https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04) and [mac](https://nodesource.com/blog/installing-nodejs-tutorial-mac-os-x/)
 
+- Install `Visual Studio` on your local machine, only for [windows](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools). (Please note that this is different from Visual Studio Code)
+- Mac user install `Xcode Command Line Tools` by running
+
+  ```git
+  xcode-select –install
+  ```
+
 - Install `pnpm` by opening your terminal with administrative (sudo) privileges and type
 
   ```git
   npm i -g pnpm
   ```
 
-- Open your terminal in your current working directory (i.e, `Cross-Platform-Activity-Tracker`) and run the following command to install all dependencies for the project
+- Open your terminal in your current working directory (i.e, `Activity-Tracker`) and run the following command to install all dependencies for the project
 
   ```pnpm
   pnpm i
   ```
 
-- Install docker on your machine. For [windows](https://docs.docker.com/desktop/windows/install/), [linux](https://docs.docker.com/engine/install/ubuntu/) and [mac](https://docs.docker.com/desktop/mac/install/)
-
 ### Step 3
 
-- Install `docker-compose` by following the [official guide](https://docs.docker.com/compose/install/)
+- Navigate to `Activity-Tracker` directory and type
 
-- Navigate to `Cross-Platform-Activity-Tracker` directory and with administrative privileges run
+  ```pnpm
+  pnpm tracker
```

Comment:
Now it should be `pnpm run dev` since Everything (desktop, server, ui) has been bundled together. (`pnpm tracker` only runs the `packages/desktop` part.)